dashboard/panels/runtime.libsonnet

```
@@ -1,15 +1,245 @@
+local grafana = import 'grafonnet/grafana.libsonnet';
+local prometheus = grafana.prometheus;
+local influxdb = grafana.influxdb;
+
 local common = import 'dashboard/panels/common.libsonnet';
+local variable = import 'dashboard/variable.libsonnet';
 
 {
   row:: common.row('Tarantool runtime overview'),
 
+  local aggregate_expr(cfg, metric_name, aggregate='sum', rate=false) =
+    local inner_expr = std.format(
+      '%s%s{%s}',
+      [
+        cfg.metrics_prefix,
+        metric_name,
+        common.prometheus_query_filters(common.remove_field(cfg.filters, 'alias')),
+      ]
+    );
+    std.format(
+      '%s(%s)',
+      [
+        aggregate,
+        if rate then std.format('rate(%s[$__rate_interval])', inner_expr) else inner_expr,
+      ]
+    ),
+
+  total_memory_per_instance(
+    cfg,
+    title='Total memory per instance',
+    description=|||
+      Total memory used by Tarantool.
```

I try to add such comments for new panels in case one would wonder "Why the panel is empty for my application"
```suggestion
      Total memory used by Tarantool.
      
      Panel minimal requirements: metrics 1.6.0.
```